compair/static/modules/assignment/assignment-module.js

```diff
@@ -909,13 +909,8 @@ module.controller("AssignmentViewController",
         // Highlight the answer if it's in the URL
         $scope.highlightAnswer = $location.search().highlightAnswer;
         if ($scope.highlightAnswer) {
-
-            $location.hash('my-answer');
+            $location.hash('answers');
             $anchorScroll();
-
-            $timeout(function() {
-                $('#my-answer').addClass('highlight-off');
```

Comment:
Since there is no more `highlight-off` class, can you add `$scope.highlightAnswer = undefined;` in the `setTab` function (so the answer will not re-highlight when switching tabs)